contracts/VotingRegistry.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+import {Paginator} from "@solarity/solidity-lib/libs/arrays/Paginator.sol";
+
+import {IVotingRegistry} from "./interfaces/IVotingRegistry.sol";
+
+/**
+ * @title VotingRegistry contract
+ */
+contract VotingRegistry is IVotingRegistry, Initializable, OwnableUpgradeable {
```

Comment:
Other contracts are UUPS. For consistency reasons I would suggest rewriting those to be UUPS as well

Reply:
Done